.devcontainer/Dockerfile

```diff
@@ -1,9 +1,15 @@
 FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.11-bullseye
 
 RUN apt-get update && apt-get install -y apt-transport-https gnupg2 procps less curl
+
 RUN wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -
 RUN echo "deb https://artifacts.elastic.co/packages/7.x/apt stable main" | tee /etc/apt/sources.list.d/elastic-7.x.list
-RUN apt-get update && apt-get install elasticsearch
+
+RUN wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -
```

Comment:
**question** why do we need to repeat this step?

Reply:
I'll undo these changes